libs/bluegl/src/BlueGLLinux.cpp

```diff
@@ -40,10 +47,17 @@ bool initBinder() {
 }
 
 void* loadFunction(const char* name) {
-    return (void*) g_driver.glXGetProcAddress((const GLubyte*) name);
+#ifdef FILAMENT_SUPPORTS_EGL_ON_LINUX
+  return (void *)eglGetProcAddress(name);
```

Comment:
`eglGetProcAddress` should be looked up like we lookup `glXGetProcAddress`

Reply:
Example on how to do this properly:

```
struct Driver {
  void* (*eglGetProcAddress)(const GLubyte*);
  void* library;
} g_driver = {nullptr, nullptr};

bool initBinder() {
  const char* library_name = "libEGL.so.1";
  g_driver.library = dlopen(library_name, RTLD_GLOBAL | RTLD_NOW);

  if (!g_driver.library) {
    return false;
  }

  g_driver.eglGetProcAddress =
      (void* (*)(const GLubyte*))dlsym(g_driver.library, "eglGetProcAddress");

  return g_driver.eglGetProcAddress;
}

void* loadFunction(const char* name) {
  return (void*)g_driver.eglGetProcAddress((const GLubyte*)name);
}

void shutdownBinder() {
  dlclose(g_driver.library);
  memset(&g_driver, 0, sizeof(g_driver));
}
```

Reply:
Makes sense, thanks for the code snippet. Fixed